resources/i18n/cs_CZ/cura.po

```diff
@@ -0,0 +1,5987 @@
+msgctxt "@label OpenGL vendor"
+msgid "<li>OpenGL Vendor: {vendor}</li>"
+msgstr "<li>OpenGL Vendor: {vendor}</li>"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:236
+#, python-brace-format
+msgctxt "@label OpenGL renderer"
+msgid "<li>OpenGL Renderer: {renderer}</li>"
+msgstr "<li>OpenGL Renderer: {renderer}</li>"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:270
+msgctxt "@title:groupbox"
+msgid "Error traceback"
+msgstr "Stopování chyby"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:356
+msgctxt "@title:groupbox"
+msgid "Logs"
+msgstr "Protokoly"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:379
+msgctxt "@title:groupbox"
+msgid ""
+"User description (Note: Developers may not speak your language, please use "
+"English if possible)"
+msgstr ""
+"Popis (Poznámka: Vývojáři nemusí mluvit vaším jazykem, pokud je to možné, "
+"použijte prosím angličtinu)"
+
+#: /home/ruben/Projects/Cura/cura/CrashHandler.py:399
+msgctxt "@action:button"
+msgid "Send report"
+msgstr "Odeslat záznam"
+
+#: /home/ruben/Projects/Cura/cura/Settings/ContainerManager.py:196
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:124
+msgctxt "@title:window"
+msgid "File Already Exists"
+msgstr "Soubor již existuje"
+
+#: /home/ruben/Projects/Cura/cura/Settings/ContainerManager.py:197
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:125
+#, python-brace-format
+msgctxt "@label Don't translate the XML tag <filename>!"
+msgid ""
+"The file <filename>{0}</filename> already exists. Are you sure you want to "
+"overwrite it?"
+msgstr ""
+"Soubor <filename> {0} </filename> již existuje. Opravdu jej chcete přepsat?"
+
+#: /home/ruben/Projects/Cura/cura/Settings/ContainerManager.py:430
+#: /home/ruben/Projects/Cura/cura/Settings/ContainerManager.py:433
+msgctxt "@info:status"
+msgid "Invalid file URL:"
+msgstr "Špatná cesta k souboru:"
+
+#: /home/ruben/Projects/Cura/cura/Settings/cura_empty_instance_containers.py:36
+msgctxt "@info:not supported profile"
+msgid "Not supported"
+msgstr "Nepodporovaný"
+
+#: /home/ruben/Projects/Cura/cura/Settings/cura_empty_instance_containers.py:55
+msgctxt "@info:No intent profile selected"
+msgid "Default"
+msgstr "Výchozí"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:136
+#, python-brace-format
+msgctxt "@info:status Don't translate the XML tags <filename> or <message>!"
+msgid ""
+"Failed to export profile to <filename>{0}</filename>: <message>{1}</message>"
+msgstr ""
+"Nepodařilo se exportovat profil do <filename> {0} </filename>: <message> {1} "
+"</message>"
+
+#: /home/ruben/Projects/Cura/cura/Settings/CuraContainerRegistry.py:143
+#, python-brace-format
+msgctxt "@info:status Don't translate the XML tag <filename>!"
+msgid ""
+"Failed to export profile to <filename>{0}</filename>: Writer plugin reported "
+"failure."
+msgstr ""
+"Export profilu do <filename> {0} </filename> se nezdařil: Zásuvný modul pro "
+"spiszápisovatele ohlásil selhání."
```

Comment:
I showed this to a Czech colleague and he claims that `spiszápisovatele` is not a proper word in Czech. Maybe there's some corrections that can be applied? In general he thinks the translation is a bit too literal.

We're currently not convinced this is a blocker so we'd merge it in anyway, but if you see some corrections here and there we can still apply them.

Reply:
Hello,
i have revisited the strings and removed some of my mistakes. The reason why it is so literal, it is just my style, it can be always more generalized.

I will look time to time into the new updates and resolve the new/not translated/poorly translated things.